modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -10,6 +10,9 @@ import 'rxjs/add/operator/map';
 import * as models                                           from '../model/models';
 import { BASE_PATH, COLLECTION_FORMATS }                     from '../variables';
 import { Configuration }                                     from '../configuration';
+{{#withInterfaces}}
+import { I{{classname}} }                                    from './{{classname}}Interface';
```

Comment:
It's embarrassing I missed that. How do we want the new naming to be exactly? Let's say in the `UserApi` case:

Class name: `UserApi`
Interface: `UserApiDefault` ?

If we can clear that out, I can make necessary changes and also fix these merge conflicts by this weekend.